### ISSUE TITLE
[Concept Entry] SQL LOOPS  in MySQl How to use loops in MySQL

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#5161]
### Reason/inspiration (optional)

We would like a new entry on the `Types of Errors` concept in SQL. The entry should go in a new file under `docs/content/sql/concepts/loops/loops.md`.     
     
The entry should include:    
    
- An introduction to the concept    
    
- A `Syntax` section that provides the syntax for the concept (if applicable)    
    
- An `Example` section that provides an example demonstrating the concept in use (if applicable)    
    
Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by AnnaN28** [comment]
@codecademy-docs , I am interested in taking this issue on. I would like to try resolve it and complete my learning program. This is my first issue so please advise where I could start.

**Comment by SaviDahegaonkar** [comment]
Hey @AnnaN28 ,
You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

Thanks  & Regards,
Savi

**Comment by AnnaN28** [comment]
Hello @SaviDahegaonkar,
Thank you for the first tips.

Yes, this is my first contribution to Codecademy Docs. I am in the process of training in the use of Linux and currently I am going through practical exercise that required selecting an one issue and working on it.

Regards, Anna

**Comment by AnnaN28** [comment]
Hello @cigar-galaxy82, @SaviDahegaonkar

I am working on this issue and I am trying to understand what the subject is?
Is it about "common errors in SQL' or "how to use loops in SQL"?

I am confused because the title of issue #5161 says "**How to use loops in SQL**" but under 'Reason/inspiration' it is written "We would like a new entry on the **Types of Errors** concept in SQL. The entry should go in a new file under docs/content/sql/concepts/loops/loops.md. "

As this is my first such kind of the exercise, I would like to understand it, prepare and describe the subject in the proper way.

Regards, Anna

**Comment by mamtawardhani** [comment]
@Sriparno08 @Radhika-okhade 
I suppose this issue should be for loops, can we have a collective opinion on this so that the issue can be edited?

**Comment by Sriparno08** [comment]
Yeah @mamtawardhani, this issue should be for loops. So, we can change `Types of Errors` to `Loops` in the issue description.

**Comment by mamtawardhani** [comment]
@AnnaN28 The issue should be for loops in SQL. Let me know if you are still working on it
